src/app/logic/ComputerPlayer.js

```diff
@@ -0,0 +1,18 @@
+import getRandomInt from '../utils/getRandomInt';
+
+export const ComputerPlayer = () => {
+  const player = {
+    askQuestion(question, callback) {
+      if (!question) throw new Error('There is no question');
+      callback(question);
+    },
+
+    getAnswer(answers, callback) {
+      if (!answers) throw new Error('There is no answers');
+      callback(answers[getRandomInt(0, 3)]);
```

Comment:
I understand we verify if answer is correct? Or send the answer to be verified? Do we store the number of correct answers or just all answers in computerPlayer object? @Enessetere